.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+    pull_request: ~
+    push:
+        branches:
+            - main
+        tags:
+            - '*'
+    schedule:
+        - cron: 0 13 * * MON,THU
+
+jobs:
+    ecs:
```

Comment:
We should also add a job that runs the unit tests for every supported PHP version I think.

Reply:
Ok, I think these are two different things: 

* ECS: Will update ECS to run with PHP 7.4. What about PHPStan? Newest PHP version or lowest supported?
* Unit tests: I copied them from the monorepo as well, but thought it was over the top. On the other hand: What's the harm? Will add them, too.

Reply:
For phpstan usually the newest I think :)